docs/source/peers/peers.md

```diff
@@ -254,101 +256,94 @@ form a blockchain network, it's worth spending a few moments understanding how
-replicated ledger.
-
-### Phase 1: Proposal
-
-Phase 1 of the transaction workflow involves an interaction between an
-application and a set of peers --- it does not involve orderers. Phase 1 is only
-concerned with an application asking different organizations' endorsing peers to
-agree to the results of the proposed chaincode invocation.
-
-To start phase 1, applications generate a transaction proposal which they send
-to each of the required set of peers for endorsement. Each of these *endorsing peers* then
-independently executes a chaincode using the transaction proposal to
-generate a transaction proposal response. It does not apply this update to the
-ledger, but rather simply signs it and returns it to the application. Once the
-application has received a sufficient number of signed proposal responses,
-the first phase of the transaction flow is complete. Let's examine this phase in
-a little more detail.
+We've seen that peers form the basis for a Fabric blockchain network, hosting
+ledgers and smart contracts which can be queried and updated by peer-connected
+applications. The mechanism by which applications and peers interact with each
+other to keep the ledger current and consistent across a channel --- the Fabric
+Gateway service --- is mediated by special nodes called *orderers*. It's to
+these orderer nodes that we now turn our attention.
+
+An ledger update transaction is different from a query transaction because a single
+peer cannot, on its own, update the ledger --- that requires the consent of other
+peers in the network, a process known as *consensus*. When the peers required to
+approve the transaction do so, and the transaction is committed to the ledger,
+the gateway service notifies the applications that the ledger has been updated.
+Peers and orderers work together to manage consensus.
+
+### Orderers and the Three Phases
+
+Previously, in the [Applications and Peers](#peers-and-application) topic, we
+saw how a client application that requests a ledger update initiates a three-phase
+process which, with help from the Fabric Gateway service, ensures that
+all peers in a Fabric network maintain current and consistent copies of the ledger.
+As described, orderers are also involved; the following sections take a closer
+look at the three-phase process from the perspective of ordering nodes and peers
+(i.e. the ordering service and the gateway service).
+
+### Phase 1: Transaction Proposal
+
+Phase 1 of the transaction workflow involves an interaction between a client
+application, peers and orderers. In Phase 1, the client application initiates
+a request to the Fabric Gateway service to evaluate a transaction proposal.
+
+The target peer, selected by the client application, executes the transaction
+by invoking chaincode --- this step can be described as simulating the transaction,
+because it runs the transaction without any effect on the ledger. The peer then
+returns its transaction result to the client.  
+
```

Comment:
The peer does not return it yet. The peer's gateway service first collects the required endorsements from other peers.